Makefile

```diff
@@ -34,16 +34,16 @@ k8s: validate
 
 .PHONY: 1.11
 1.11:
-	$(MAKE) k8s kubernetes_version=1.11.10 kubernetes_build_date=2019-08-14
+	$(MAKE) k8s kubernetes_version=1.11.10 kubernetes_build_date=2019-12-13
```

Comment:
this one should be `1.11.10  2019-09-27`. 1.11 is on deprecate path, we didn't have new build for 1.11.10